CMakeLists.txt

```diff
@@ -9,8 +9,8 @@ SET(CMAKE_C_STANDARD 11)
 
 SET(CMAKE_C_FLAGS "-std=c99 -Wall -O2 -g ${CC_WARNING_FLAGS} ${CMAKE_C_FLAGS}")
 
-if((NOT ${DISABLE_DEBUG_PRINTF}) OR (NOT DEFINED DISABLE_DEBUG_PRINTF))
-    SET(CMAKE_C_FLAGS "-DENABLE_DEBUG_PRINTF ${CMAKE_C_FLAGS}")
+if((${DISABLE_DEBUG_PRINTF}) AND (DEFINED DISABLE_DEBUG_PRINTF))
```

Comment:
I think that can be simplified to just `$DISABLE_DEBUG_PRINTF`.

Reply:
As in "if ($DISABLE_DEBUG_PRINTF)  SET(...." ?